frontend/catalyst/api_extensions/callbacks.py

```diff
@@ -185,9 +185,26 @@ def getOperandTypes(self):
         return map(type, self.getOperands())
 
 
+def clear_callback_cache():
```

Comment:
Move to class method

Reply:
https://github.com/PennyLaneAI/catalyst/pull/782/commits/ece6c70e7d205e507abe53741c0943a84a9d7486